pvactools/lib/run_argument_parser.py

```diff
@@ -111,6 +111,12 @@ def __init__(self, tool_name, input_file_help):
             default=1,
             help="Number of threads to use for parallelizing peptide-MHC binding prediction calls.",
         )
+        parser.add_argument(
+            "--netmhc_pan_version",
```

Comment:
You'll want to name this argument `--netmhcii-pan-version` (or `--netmhc-ii-pan-version` or `--netmhciipan-version`; not sure which is better - might want to put it to a vote on Slack) to avoid confusion since there is a different class I algorithm NetMHCpan. Additionally, using dash instead of underscore is the standard in Python for variant names. The parser automatically changes the parsed variable to use underscores.